src/mslice/presenters/slice_plotter_presenter.py

```diff
@@ -20,18 +19,16 @@ class SlicePlotterPresenter(PresenterUtility):
     def __init__(self):
         self._main_presenter = None
         self._slice_cache = {}
-        self._sample_temp_fields = []
 
     def plot_slice(self, selected_ws, x_axis, y_axis, intensity_start, intensity_end, norm_to_one, colourmap):
         workspace = get_workspace_handle(selected_ws)
         self.create_slice(workspace, x_axis, y_axis, intensity_start, intensity_end, norm_to_one, colourmap)
         self.plot_from_cache(workspace)
 
     def create_slice(self, selected_ws, x_axis, y_axis, intensity_start, intensity_end, norm_to_one, colourmap):
-        sample_temp = sample_temperature(selected_ws, self._sample_temp_fields)
         norm = Normalize(vmin=intensity_start, vmax=intensity_end)
         slice = compute_slice(selected_ws, x_axis, y_axis, norm_to_one)
-        self._cache_slice(slice, colourmap, norm, sample_temp, x_axis, y_axis)
+        self._cache_slice(slice, colourmap, norm, None, x_axis, y_axis)
```

Comment:
Would it be possible to check with the users that caching the sample temperature is no longer wanted before we remove it?

Reply:
This feature was accidently removed a couple of months ago, so I think we're safe to remove. I'll raise an issue to investigate whether this should be reimplemented. At this point it's more effort to fix than to remove.

Reply:
Yes, I think it's fine to remove it. In practice different slices often have different temperatures so the caching is not always useful. 

Is `_cache_slice` referenced elsewhere where it needs the temperature? If not, maybe it can be refactored to remove this argument, instead of passing `None`?

Ideally what we could have in future is to cache the name of the log variable which holds the temperature and use that to determine the temperature of the slice. However (1) this was never implemented, and more importantly (2) the `.nxspe` file format we often use discards all the log information :( the reduced/processed `.nxs` files retains them but it's hard to get users to change to using it...

Reply:
It is not used elsewhere. I think I will leave it as is now, and implement the feature as you outline shortly.